xbmc/ContextMenuItem.cpp

```diff
@@ -50,9 +50,13 @@ bool CContextMenuItem::Execute(const CFileItemPtr& item) const
   if (!CServiceBroker::GetAddonMgr().GetAddon(m_addonId, addon))
     return false;
 
+  bool reuseLanguageInvoker = false;
+  if (addon->ExtraInfo().find("reuselanguageinvoker") != addon->ExtraInfo().end())
+    reuseLanguageInvoker = addon->ExtraInfo().at("reuselanguageinvoker") == "true";
+
```

Comment:
Move the `reuselanguageinvoker` stuff into a separate commit.

Reply:
Done.